src/web-auth/captcha.js

```diff
@@ -276,7 +275,19 @@ function handleCaptchaProvider(element, options, challenge) {
       var global = globalForCaptchaProvider(challenge.provider);
       if (challenge.provider === ARKOSE_PROVIDER) {
         var retryCount = 0;
+        var arkoseLoaded = false;
         arkose.setConfig({
+          onReady: function () {
+            if (!arkoseLoaded) {
+              done(null, {
+                triggerCaptcha: function (solvedCallback) {
+                  arkose.run();
+                  captchaSolved = solvedCallback;
+                }
+              });
+              arkoseLoaded = true;
+            }
+          },
```

Comment:
I thought about also calling done on onError but decided against it since we have a retry mechanism. But thinking about this again, it makes sense if Arkose fails the maximum retries and still hasn't been to loaded to call done inside onError.

Reply:
> Was reading docs, and see we call arkose.reset() on error. Is the assumption that Arkose will re-fire the onReady event after an error + reset operation?

Never mind I see what you're saying. I'm not sure on the order of operations for the callbacks. If an error occurs again during reset after retrying, I wonder if it'll fire onReady callback again due to the reset or just fire the onError callback.